src/queue.ts

```diff
@@ -97,7 +98,16 @@ export class AmqpQueue<T> extends EventEmitter {
       content = new Buffer(json);
     }
 
-    chnl.sendToQueue(this.options.name, content, opts);
+    const exchange = options.exchangeOverride === '' || options.exchangeOverride ?
```

Comment:
Wouldn't this always apply `options.exchangeOverride`? It defaults as `''` so this would always be true.

Reply:
`options.exchangeOverride` is the options passed into the `publish` method, which defaults as `{}`

Reply:
Fair enough, got the `exchange` on line 12 mixed up with `exchangeOverride` (yay work travel)
LGTM! Thanks again!